FlagsmithClient/src/test/java/com/flagsmith/entities/TraitEntityTests.kt

```diff
@@ -77,7 +77,7 @@ class TraitEntityTests {
 
     @Test
     fun testTraitConstructorStringType() {
-        val trait = Trait( "string-key", "string-value")
+        val trait = Trait( "string-key", "string-value", false)
```

Comment:
As I mentioned above, this shouldn't be necessary. This shouldn't need to be a breaking change.

Reply:
You're right, Corrected